src/ResourceManagement/HDInsight/HDInsight/Microsoft.Azure.Management.HDInsight.nuget.proj

```diff
@@ -2,8 +2,8 @@
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ItemGroup>
     <SdkNuGetPackage Include="Microsoft.Azure.Management.HDInsight">
-      <PackageVersion>1.3.0</PackageVersion>
+      <PackageVersion>2.0.0</PackageVersion>
```

Comment:
These are throwing because you sepcified the second attribute as 'true', see:

For details on the absolete attribute, see: https://msdn.microsoft.com/en-us/library/22kk2b44(v=vs.90).aspx

If these properties no longer work then this is a breaking change.  You will need to have a p;lan to mitigate this break in azure powershell if you plan to use that library there.  Please acknowledge and then this issue will be resolved.